src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -319,13 +323,13 @@ public override bool Execute ()
 					foreach (var ca in resolver.GetAssembly (assemblyItem.ItemSpec).CustomAttributes) {
 						switch (ca.AttributeType.FullName) {
 						case "Android.IncludeAndroidResourcesFromAttribute":
-							AddAttributeValue (androidResources, ca, "XA5206", "{0}. Android resource directory {1} doesn't exist.", true, fullPath);
+							AddAttributeValue (androidResources, ca, "XA5206", "{0}. (Details: Android resource directory {1} doesn't exist.)", true, fullPath);
```

Comment:
All the messages are not great. :-)

The problem is twofold: People don't read messages, and the IDE will truncate messages. We should thus move the "useful" data to the *beginning*:

> error XA5207: Could not find `foo.jar` while restoring resources for package `Foo`. The authors of the package should set the SourceUrl or InstallInstructions property on Java.Interop.JavaLibraryReferenceAttribute.

This moves the relevant/important message to the front, and the suggestion for package authors to the end. It's also shorter (somewhat...).

> So maybe it would be best to split the specific file information into its own error line, if that wouldn't break convention?

That *could* be better (theoretically), but *I* don't see how that's an improvement *in this circumstance*.